packages/medusa/src/repositories/payment-collection.ts

```diff
@@ -0,0 +1,6 @@
+import { PaymentCollection } from "./../models/payment-collection"
+import { EntityRepository, Repository } from "typeorm"
+
+@EntityRepository(PaymentCollection)
+// eslint-disable-next-line max-len
```

Comment:
Suggestion: i ve seen that in multiple places by my linter does not raise an error if i remove it. It is using the one from the project

Reply:
That's odd, because if you check
"export class PaymentCollectionRepository extends Repository<PaymentCollection> {}".length

has 81 chars, and the limit is 80. It should flag it for you

Reply:
Ahah i might have miss read then 🤣